packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.test.tsx

```diff
@@ -146,36 +146,6 @@ describe('VitalsBiometricsForm: ', () => {
 
     await waitFor(() => user.click(saveButton));
 
-    expect(mockSavePatientVitals).toHaveBeenCalledTimes(1);
```

Comment:
Why did you remove these tests, we need to test that the form submits successfully.